eth/v1alpha1/validator.proto

```diff
@@ -15,13 +15,120 @@ syntax = "proto3";
 
 package ethereum.eth.v1alpha1;
 
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+import "eth/v1alpha1/block.proto";
+import "eth/v1alpha1/attestation.proto";
+
 option csharp_namespace = "Ethereum.Eth.v1alpha1";
 option go_package = "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1;eth";
 option java_multiple_files = true;
 option java_outer_classname = "ValidatorProto";
 option java_package = "org.ethereum.eth.v1alpha1";
 option php_namespace = "Ethereum\\Eth\\v1alpha1";
 
+// Beacon node validator API
+//
+// The beacon node validator API enables a validator to connect
+// and perform its obligations on the Ethereum 2.0 phase 0 beacon chain.
+service BeaconNodeValidator {
+    // Retrieves validator duties for the requested validators.
+    //
+    // The server returns a list of duties which are the actions should be performed by validators for a given epoch.
```

Comment:
Can we mention briefly what those duties are? There are only two and are unambiguous. Important to clarify so that people don't think there are more.

Reply:
The word polling makes me think we need a reorg stream. Always seems wasteful to poll when you have streaming built in.

This can be addressed in later issue